data/test/scenarios/wml_tests/ScenarioWML/EventWML/events-test_filters.cfg

```diff
@@ -198,6 +198,25 @@
         # Make sure the attacks hit
         {FORCE_CHANCE_TO_HIT (id=bob) (id=alice) 100 ()}
         {FORCE_CHANCE_TO_HIT (id=alice) (id=bob) 100 ()}
+        [object]
```

Comment:
This should be a new unit test, because there are other ways that `event_test_filter_attack` could break. While those would likely make this test fail too, it's worth testing the general functionality first.

One possible bug would be if `[filter_attack]` only matched for attacks with specials. Having two separate tests would catch that, but this single combined test wouldn't.

Reply:
added separate test